### ISSUE TITLE
Sky-page-summary-title has empty headers with is causing accessibility tests to fail 

### ISSUE THREAD

**Issue by Blackbaud-VeehaKhanna** [blackbaud/skyux-layout#165]
### Expected behavior
No error is presented with default Page Summary.

### Actual behavior
```
aXe - [Rule: 'empty-heading'] Headings must not be empty - WCAG: best-practice
               Get help at: https://dequeuniversity.com/rules/axe/3.5/empty-heading?application=axeAPI
               Elements:
        <h1 _ngcontent-a-c66="" class="sky-page-summary-title sky-page-heading"><div _ngcontent-a-c57="" class="sky-page-heading" data-bbauto-field="email"></div></h1>

```
### Steps to reproduce
Html 
```
 <sky-page-summary-title>
              <div class="sky-page-heading" data-bbauto-field="email">{{profile.email}}</div>
 </sky-page-summary-title>

```
Test 
```
 it('Shows the popover icon if it is apple anonymized user', async(() => {
    spyOn(profileSvc, 'profileGet').and.returnValue(observableOf<any>({
       permissions: {},
       use_mfa: false,
       provider: 'apple',
       email: 'test@privaterelay.appleid.com'
    }));

    fixture.componentInstance.socialUser = true;
    fixture.componentInstance.providerName = 'Apple';
    fixture.componentInstance.anonymousEmail = true;
    fixture.detectChanges();
    testHelper.elementHasText('apple-anonymous-user', "What's this email address?");
    testHelper.elementHasText('profile-not-allowed-change-password-social',
    "You can't edit your password when using Apple authentication.");
    expect(fixture.nativeElement).toBeAccessible();
  }));

```
### Severity
Low
### Impact
Low

**Comment by Blackbaud-VeehaKhanna** [comment]
That was it, thanks @Blackbaud-SteveBrush. I'll fix that.

**Comment by Blackbaud-VeehaKhanna** [comment]
Sorry about the back and forth, that fix worked for that test but not this one. @Blackbaud-SteveBrush 

```
it('Catches an error in ngOnInit when attempting to get the profile', async(() => {
    spyOn(profileSvc, 'profileGet').and.callFake(
      TestHelper.createErrorResponseFunction(500, 'Internal Server Error')
    );
    let router = de.injector.get(Router);
    let routerSpy = spyOn(router, 'navigate').and.stub();

    fixture.detectChanges();
    fixture.whenStable().then(() => {
      expect(routerSpy.calls.first().args[0]).toEqual(['/error']);
    });
    fixture.whenStable().then(() => {
      expect(fixture.nativeElement).toBeAccessible();
    });
  }));
```

**Comment by Blackbaud-VeehaKhanna** [comment]
I had 9 tests failing with the above error and adding whenStable().then() fixed 5 of them.

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-VeehaKhanna - it appears that this is an issue when the `profile.email` is undefined. You are still rendering the HTML for the title but then not passing anything into it. So the header renders but has no internal content. To get around this you should be able to place an `*ngIf="profile.email"` on the `sky-page-summary-title` element.

**Comment by Blackbaud-VeehaKhanna** [comment]
Gotcha I needed to be more explicit in my tests and set the email through `fixture.componantInstance.profile.email`. Thanks @Blackbaud-SteveBrush Done for real now.
